### ISSUE TITLE
Could not find gem 'refinerycms-i18n'

### ISSUE THREAD

**Issue by CJBrew** [refinery/refinerycms#2835]
There seems to be something wrong with the engine generator. EITHER THAT or I'm doing something silly...

I get the following error on running `bundle install` in the extension folder:
__Could not find gem 'refinerycms-i18n (~> 3.0.0) ruby', which is required by gem
'refinerycms-things (>= 0) ruby', in any of the sources.__

Script to reproduce
```bash
rvm install 2.1.5
rvm 2.1.5@refinery_rails419 --create

gem install rails --version 4.1.9
gem install execjs

rails new TestApp -m http://refinerycms.com/t/edge
cd TestApp/

bundle install

rake db:drop db:create db:migrate db:seed

spring stop # because spring makes rails g hang...

rails g refinery:engine thing info:string

cd vendor/extensions/things
bundle install
```
Std Output:

```text
<LOTS CUT>
      create  vendor/extensions/things/tasks/rspec.rake
      create  vendor/extensions/things/tasks/testing.rake
      create  vendor/extensions/things/spec/support/factories/refinery/things.rb
      create  vendor/extensions/things/spec/features/refinery/things/admin/things_spec.rb
      create  vendor/extensions/things/spec/spec_helper.rb
      create  vendor/extensions/things/spec/models/refinery/things/thing_spec.rb
      append  Gemfile
------------------------
Now run:
bundle install
rails generate refinery:things
rake db:migrate
rake db:seed
Please restart your rails server.
------------------------
Updating https://github.com/refinery/refinerycms.git
Fetching gem metadata from https://rubygems.org/.........
Fetching additional metadata from https://rubygems.org/..
Resolving dependencies...
Could not find gem 'refinerycms-i18n (~> 3.0.0) ruby', which is required by gem
'refinerycms-things (>= 0) ruby', in any of the sources.
```

**Comment by CJBrew** [comment]
Full output file https://gist.github.com/CJBrew/7e6a0468f12ae2884fca

**Comment by parndt** [comment]
Fixed by releasing refinerycms-i18n 3.0.0
